Fix Random CMake Error

| | |
|:---|:---|
|**_F´ Version_**| |
|**_Affected Component_**|   |
---
## Problem Description

Fix the random cmake error that keeps poping up in CI.

## Context / Environment

CI

## How to Reproduce

???

## Expected Behavior

No crash TM

See https://github.com/nasa/fprime/actions/runs/23546634911/job/68548774744?pr=4890

```
/home/runner/work/fprime/fprime/Svc/PassThroughRouter/test/ut/PassThroughRouterTester.cpp:9:10: fatal error: Svc/FprimeRouter/FprimeRouterGTestBase.hpp: No such file or directory
    9 | #include "Svc/FprimeRouter/FprimeRouterGTestBase.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Seems there may be some race condition in the generation of files. That being said I'm extremely surprised that `PassThroughRouterTester.cpp` would try to include `FprimeRouterGTestBase.hpp` ?

Thanks to @Kronos3 for catching the info

--- a/Svc/PassThroughRouter/test/ut/PassThroughRouterTester.cpp
+++ b/Svc/PassThroughRouter/test/ut/PassThroughRouterTester.cpp
@@ -5,8 +5,6 @@
 // ======================================================================
 
 #include "PassThroughRouterTester.hpp"
-#include "PassThroughRouterGTestBase.hpp"
-#include "Svc/FprimeRouter/FprimeRouterGTestBase.hpp"
 
 namespace Svc {
 

